myPortfolio/style.css

```diff
@@ -1,40 +1,116 @@
+/* san serif */
 
-/* .photoAndIntro {
-    display: flex;
+html {
+    /* background-color: rgb(255, 230, 208); */
+    /* light blue, skyblue, cream, beige!, pale yellow, peach! coral */
+    /* background-color: #ffffff; */
+    background-color: lavender;
 }
 
-.containers {
-    flex: 1;
-} 
+.bold-text {
+    font-weight: bold;
+  }
+
+body {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    font-family: "Amatic SC", sans-serif;
+
+}
 
 img {
     max-width: 100%;
-  height: auto;
-} */
+    height: auto;
+    border: 10px dotted lightgoldenrodyellow;
+}
 
+.headings {
+    font-weight: bold;
+    font-size: larger;
+}
 
+#Bio {
+    text-align: center;
+    font-family: "Amatic SC", sans-serif;
+    margin-left: 35px;
+    padding-top: 15px;
 
+}
 
-/* 
-.containers {
-    display: inline-block;
-    width: 48%; Adjust the width as needed
-    vertical-align: top; Align containers at the top
+/* From Google Fonts */
+@import url('https://fonts.googleapis.com/css2?family=Italianno&display=swap');
+/* Apply the font to the h1 element */
+h1 {
+  font-family: 'Italianno', cursive;
+  color: #816f0e;
+  margin-bottom: 0;
+}
+
+h2 {
+    margin-top: 0;
+    font-family: 'Italianno', cursive;
+    color: #816f0e;
+}
```

Comment:
Not really important for now, but when you start writing CSS for a design system or when you're working on a much larger website, you can do stuff like

```h1, h2 {
    font-family: 'Italianno', cursive;
    color: #816f0e;
}

h1 {
   margin-bottom: 0;
}

h2 {
   margin-top: 0;
}
```

so you can group the classes and/or tags together if they implement the same styles.

Reply:
And you can even take it a step further since you're using `margin-top: 0` and `margin-bottom: 0` in multiple places...
```
.noMarginTop {
   margin-top: 0
}
```

And then add this class to the different elements that are using it. 
```
<h2 class="noMarginTop">... 
<h5 id="h5Two" class=noMarginTop">...
```